src/test/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserPostDaoITest.java

```diff
@@ -0,0 +1,41 @@
+package com.vinylteam.vinyl.dao.jdbc;
+
+import com.vinylteam.vinyl.dao.UserPostDao;
+import com.vinylteam.vinyl.entity.UserPost;
+import com.vinylteam.vinyl.util.DataGeneratorForTests;
+import com.vinylteam.vinyl.util.DatabasePreparerForITests;
+import org.junit.jupiter.api.*;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class JdbcUserPostDaoITest {
+    private final DatabasePreparerForITests databasePreparer = new DatabasePreparerForITests();
```

Comment:
For future. Please add empty row after interface declaration and before closing it. It is kind of our code style